means.py

```diff
@@ -22,10 +22,11 @@ def mean_rotor_in_chordal_metric(R, t=None):
     faster).
 
     """
-    if not t:
-        return np.quaternion(*(np.sum(as_float_array(R)))).normalized()
+    R_as_array = np.asarray(R, dtype=np.quaternion).view((np.double, 4))
```

Comment:
You should just import the function, rather than reproducing it; you never know if I'll discover some better way to do that.
```python
from . import as_float_array
```